api/main.go

```diff
@@ -17,12 +13,5 @@ func recoverWith(f func(err error)) {
 }
 
 func main() {
-	// prime the API and instance for builds by logging into private registries
-	// and pulling down latest app images
-	go func() {
-		models.LoginPrivateRegistries()
-		models.PullAppImages()
```

Comment:
It's handled by Docker. Pulling the images on the API instance doesn't necessarily help anymore anyway.